src/main/java/com/dreammaster/gthandler/GT_Loader_Wires.java

```diff
@@ -16,29 +16,30 @@ public void run() {
 
         // Takes 20!!! slots per wire
-        makeWires(Materials.ElectricalSteel, 11490, 1L, 2L, 2L, GTValues.V[2], true, false);
-        makeWires(Materials.EnergeticAlloy, 11510, 2L, 4L, 2L, gregtech.api.enums.GTValues.V[3], true, false);
-        makeWires(Materials.VibrantAlloy, 11530, 2L, 4L, 4L, gregtech.api.enums.GTValues.V[4], true, false);
-        makeWires(Materials.MelodicAlloy, 11550, 2L, 4L, 4L, gregtech.api.enums.GTValues.V[5], true, false);
-        makeWires(Materials.StellarAlloy, 11570, 4L, 8L, 6L, gregtech.api.enums.GTValues.V[6], true, false);
-        makeWires(Materials.Trinium, 11450, 4L, 8L, 6L, GTValues.V[7], true, false);
-        makeWires(Materials.ElectrumFlux, 1900, 1L, 2L, 3L, GTValues.V[8], true, false);
-        makeWires(Materials.Bedrockium, 11310, 1L, 32L, 2L, GTValues.V[9], true, false);
-        makeWires(Materials.Osmiridium, 11610, 1L, 2L, 8L, GTValues.V[6], true, false);
-        makeWires(Materials.HSSE, 11590, 2L, 4L, 6L, GTValues.V[6], true, false);
-        makeWires(Materials.HSSS, 11470, 4L, 8L, 6L, GTValues.V[9], true, false);
-        makeWires(Materials.Draconium, 11330, 4L, 32L, 8L, GTValues.V[10], true, false);
-        makeWires(Materials.NetherStar, 11350, 4L, 16L, 4L, GTValues.V[11], true, false);
-        makeWires(Materials.Quantium, 11370, 4L, 16L, 2L, GTValues.V[12], true, false);
-        makeWires(Materials.BlackPlutonium, 11390, 4L, 16L, 1L, GTValues.V[13], false, false);
-        makeWires(Materials.DraconiumAwakened, 11410, 4L, 16L, 1L, GTValues.V[14], false, false);
+        makeWires(Materials.ElectricalSteel, 11490, 1L, 2L, 3L, GTValues.V[2], true, false);
+        makeWires(Materials.EnergeticAlloy, 11510, 1L, 2L, 4L, gregtech.api.enums.GTValues.V[3], true, false);
+        makeWires(Materials.VibrantAlloy, 11530, 3L, 6L, 6L, gregtech.api.enums.GTValues.V[4], true, false);
+        makeWires(Materials.EndSteel, 1860, 3L, 6L, 6L, gregtech.api.enums.GTValues.V[5], true, false);
+        makeWires(Materials.MelodicAlloy, 11550, 2L, 4L, 6L, gregtech.api.enums.GTValues.V[6], true, false);
+        makeWires(Materials.StellarAlloy, 11570, 12L, 24L, 12L, gregtech.api.enums.GTValues.V[8], true, false);
+        makeWires(Materials.Trinium, 11450, 3L, 6L, 6L, GTValues.V[7], true, false);
+        makeWires(Materials.ElectrumFlux, 1900, 3L, 2L, 8L, GTValues.V[8], true, false);
+        makeWires(Materials.Bedrockium, 11310, 6L, 48L, 3L, GTValues.V[9], true, false);
+        makeWires(Materials.Osmiridium, 11610, 1L, 2L, 16L, GTValues.V[7], true, false);
+        makeWires(Materials.HSSE, 11590, 4L, 8L, 6L, GTValues.V[6], true, false);
+        makeWires(Materials.HSSS, 11470, 4L, 8L, 8L, GTValues.V[9], true, false);
+        makeWires(Materials.Draconium, 11330, 32L, 96L, 12L, GTValues.V[10], true, false);
+        makeWires(Materials.NetherStar, 11350, 64L, 160L, 4L, GTValues.V[11], true, false);
```

Comment:
Nether star is a very expensive material but a very good wire if you spend a lot of effort to increase it. When it's used, this loss hinders it quite a bit imo which I don't think is a great change

Reply:
The point of it was that it's a UIV cable, and renewable in _EV_ That is, what, 10 tiers of influence? Imo its warranted.

Reply:
Discussed on discord, will adjust to 16 loss rather than 64

https://discord.com/channels/181078474394566657/401118216228831252/1317562056068567051